include/flash_map_pm.h

```diff
@@ -40,6 +41,6 @@
 	UTIL_CAT(PM_, UTIL_CAT(UTIL_CAT(PM_, UTIL_CAT(PM_ID(label), _LABEL)), _SIZE))
 
 #define FLASH_AREA_LABEL_EXISTS(label) \
-	IS_ENABLED(FLASH_AREA_SIZE(label))
+	IS_ENABLED(PM_IS_ENABLED(label))
```

Comment:
@hakonfam Didn't we talk about this on a different branch yesterday? Was it the same solution?

Reply:
Note that this and a bunch of other changes come from #3357 which this is based on. Look at the last commits to see the changes that properly belong to the PR.